.gitignore

```diff
@@ -10,7 +10,7 @@
 /addons/*/*
 /tests/*
 /node_modules
-
+addons/
```

Comment:
I think better solution is to rename `/addons/.gitkeep` to `/addons/.gitignore` with content

```
*
!.gitignore 
```